### PR TITLE
Add teleoperation of manipulator from joystick

### PR DESCRIPTION
**Pull request by knord94**

### CODE REVIEW

**Comment by aestene** [comment]
This pull request has automatically been marked as stale as there has been no activity for 30 days. Remove stale label or comment or this will be closed in 7 days.

**Comment by knord94** [comment]
Will finish this after Easter.

**Comment by knord94** [comment]
> This pull request has automatically been marked as stale as there has been no activity for 30 days. Remove stale label or comment or this will be closed in 7 days.

Please do not close this issue.

File: ros_packages/isar_turtlebot_moveit_config/config/chomp_planning.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+smoothness_cost_jerk: 0.0
+ridge_factor: 0.01
+use_pseudo_inverse: false
+pseudo_inverse_ridge_factor: 1e-4
+joint_update_limit: 0.1
+collision_clearence: 0.2
+collision_threshold: 0.07
+use_stochastic_descent: true
+enable_failure_recovery: true
+max_recovery_attempts: 5
```
Comment (Christdej) [inline ros_packages/isar_turtlebot_moveit_config/config/chomp_planning.yaml:18]: Add newline

File: ros_packages/isar_turtlebot_moveit_config/config/fake_controllers.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+controller_list:
+  - name: fake_arm_controller
+    joints:
+      - joint1
+      - joint2
+      - joint3
+      - joint4
+  - name: fake_gripper_controller
+    joints:
+      - gripper
```
Comment (Christdej) [inline ros_packages/isar_turtlebot_moveit_config/config/fake_controllers.yaml:10]: Add newline

File: ros_packages/isar_turtlebot_moveit_config/config/joint_limits.yaml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  joint3:
+    has_velocity_limits: true
+    max_velocity: 4.8
+    has_acceleration_limits: false
+    max_acceleration: 0
+  joint4:
+    has_velocity_limits: true
+    max_velocity: 4.8
+    has_acceleration_limits: false
+    max_acceleration: 0
```
Comment (Christdej) [inline ros_packages/isar_turtlebot_moveit_config/config/joint_limits.yaml:34]: Add newline

File: ros_packages/isar_turtlebot_moveit_config/config/ompl_planning.yaml [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+    - BiTRRT
+    - LBTRRT
+    - BiEST
+    - ProjEST
+    - LazyPRM
+    - LazyPRMstar
+    - SPARS
+    - SPARStwo
+  projection_evaluator: joints(gripper,gripper_sub)
+  longest_valid_segment_fraction: 0.005
```
Comment (Christdej) [inline ros_packages/isar_turtlebot_moveit_config/config/ompl_planning.yaml:178]: Add newline

File: ros_packages/isar_turtlebot_moveit_config/config/sensors_3d.yaml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# The name of this file shouldn't be changed, or else the Setup Assistant won't detect it
+sensors:
+  - {}
```
Comment (Christdej) [inline ros_packages/isar_turtlebot_moveit_config/config/sensors_3d.yaml:3]: Add newline

File: ros_packages/isar_turtlebot_moveit_config/launch/gdb_settings.gdb [hunk 6]
```diff
@@ -0,0 +1,8 @@
+## This file allows developers to set breakpoints in the move_group.launch file
+## To set a breakpoint, compile with debug flag set to true. Add a line below with the source file name,
+## a colon, then the line number to break on. Then launch move_group with argument debug:=true
+
+set breakpoint pending on
+
+## Example break point:
+#break trajectory_execution_manager.cpp:228
```
Comment (Christdej) [inline ros_packages/isar_turtlebot_moveit_config/launch/gdb_settings.gdb:8]: Add newline

**Review by Christdej — CHANGES_REQUESTED** [review]

**Review by Christdej — APPROVED** [review]

**Review by Christdej — APPROVED** [review]
LGTM

**Pull request merged**